Modulo_4/exercise/4_5/exercícios.html

```diff
@@ -0,0 +1,126 @@
+
+      //Adicione a tag div com a classe center-content como filho da tag div criada no passo 2;
+      let centerContent = document.createElement("div");
+      centerContent.className = 'center-content';
+      document.getElementsByClassName('main-content')[0].appendChild(centerContent);
+
+      //Adicione a tag p como filho do div criado no passo 3 e coloque algum texto;
+      let paragrafoOne = document.createElement("p");
+      paragrafoOne.innerHTML = "Adicionando um texto para concluir o exercício!";
+      document.getElementsByClassName("center-content")[0].appendChild(paragrafoOne);
+
+      //Adicione a tag div com a classe left-content como filho da tag div criada no passo 2;
+      let leftContent = document.createElement("div");
+      leftContent.className = 'left-content';
+      document.getElementsByClassName("main-content")[0].appendChild(leftContent);
+
+      //Adicione a tag div com a classe right-content como filho da tag div criada no passo 2;
+      let rightContent = document.createElement('div');
+      rightContent.className = "right-content";
+      document.getElementsByClassName("main-content")[0].appendChild(rightContent);
+
+      //Adicione uma imagem com src configurado para o valor https://picsum.photos/200 e classe small-image. Esse elemento deve ser filho do div criado no passo 5
+      let firstImg = document.createElement("img");
+      firstImg.src = "https://picsum.photos/200";
+      firstImg.className = "small-image"
+      document.getElementsByClassName("left-content")[0].appendChild(firstImg);
+
+      //Adicione uma lista não ordenada com os valores de 1 a 10 por extenso, ou seja, um, dois, três, … como valores da lista. Essa lista deve ser filha do div criado no passo 6;
+      let lista = document.createElement("ul");
+      let numbers = ['um','dois','tres','quatro','cinco','seis','sete','oito','nove','dez'];
+      document.getElementsByClassName("right-content")[0].appendChild(lista)
+      for (let i in numbers){
+        let itens = document.createElement("li");
+        itens.innerHTML = numbers[i];
+        lista.appendChild(itens)
+      }
+
+      //Adicione 3 tags h3, todas sendo filhas do div criado no passo 2.
+        for (let i=0; i<3; i++){
+          let tagH3 = document.createElement("h3");
+          document.getElementsByClassName("main-content")[0].appendChild(tagH3);
+        }
+
+        //PARTE II
+        document.getElementsByTagName('h1')[0].className = "title";
+        for (let i=0; i<3; i++){
+        document.getElementsByTagName('h3')[i].className = "description"
+        }
+        leftContent.parentNode.removeChild(leftContent);
+        rightContent.style.marginRight = "auto";
+        centerContent.style.backgroundColor = "green";
+        var ultimo = document.getElementsByTagName('li')[9];
+        var penultimo = document.getElementsByTagName('li')[8];
+        ultimo.parentNode.removeChild(ultimo);
+        penultimo.parentNode.removeChild(penultimo);
```

Comment:
Assumindo "lista" como sua "ul"
```suggestion
        lista.removeChild(lista.lastChild);
        lista.removeChild(lista.lastChild);
```